example-apps/react-js/src/pages/Editor/index.jsx

```
@@ -1,21 +1,24 @@
+import { useLocation } from "react-router-dom";
+import EditorControls from "./EditorControls";
+import EditorHelmet from "./EditorHelmet";
 
 import styles from "./Editor.module.css";
 
 const SHAPE_SELECTOR = "path,rect,circle,polygon,ellipse";
 
 export default function Editor() {
   const { state } = useLocation();
-  const { svg } = state || {};
+  const { svg: initialSvg, fileData, originalSrc, imgBilateralFiltered, initialSettings } = state || {};
```

ESLint

[error] 'originalSrc' is assigned a value but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)